typescript/packages/subsurface-viewer/src/storybook/SubsurfaceViewer.stories.jsx

```diff
@@ -644,3 +681,55 @@ ViewMatrixMargin.args = {
         ],
     },
 };
+
+export const IsLoadedCallback = (args) => {
```

Comment:
Please move to `SubsurfaceViewer.stories.tsx`. Let's try to deprecate the JSX stories.